### DISCUSSION TITLE
how to obtain thenaxcakue's record before show in a form Text Input?

### DISCUSSION THREAD

**Discussion by karintemi** [filamentphp/filament#3724] [Help] [answered]
when i create a new invoice_number from table invoices, i need to obtain the max vakue and add one, to show in a form like tect input as disabled().
whats te best way to do it}
Thanks a lot

**Comment by danharrin** [comment, marked as answer]
Use `$get('../../repeater_name')` inside a `default(fn ($get) => ...)`, loop through the repeater items array and get the max
